CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 
 - Add `time_nanos` to `BlockInfo` allowing access to high precision block times.
 - Change `FullDelegation::accumulated_rewards` from `Coin` to `Vec<Coin>`.
+- Rename `InitResponse::log`, `MigrateResponse::log` and `HandleResponse::log`
+  to `InitResponse::attributes`, `MigrateResponse::attributes` and
+  `HandleResponse::attributes`.
+- Rename `LogAttribute` to `Attribute`.
+- Remove `log` in favour of `Attribute::new`.
+- Rename `Context::add_log` to `Context::add_attribute`.
```

Comment:
I would go for `Context::emit`, which is very clear to anyone who touched Ethereum. `add_attribute` may make sense to cosmos-sdk devs, but not much more.

Reply:
There are two issues I have with "emit":

1. We are not emitting multiple events when calling the function multiple times. [We are adding attributes to a single event](https://github.com/CosmWasm/wasmd/blob/v0.10.0/x/wasm/internal/types/types.go#L245) that is emitted. This is prettry close to Etherem: 
![Screenshot_2020-09-09 Contracts — Solidity 0 5 3 documentation](https://user-images.githubusercontent.com/2603011/92565930-c2b89f80-f27b-11ea-9e7c-6c81b675b50a.png) We could can change the framework to support multiple events per contract execution. But this would be a bigger change. Here I try to make clear what is happening so far.
2. In contrast to the `emit` keyword in solidity, we don't know if the context will ever be used for something. So I find it misleading to call it emitting when you just store it in an object that is potentially returned. If we want to have an solidity style imperative framkework, we should get some semi-global object (somewhere close to API) that does the emitting of messages and events instead of composing a return value.